### PR TITLE
`VariadicSplitLayerTest` refactoring to API2.0

### PR DESCRIPTION
**Pull request by vurusovs**

### CODE REVIEW

File: src/plugins/intel_cpu/tests/functional/shared_tests_instances/single_layer_tests/variadic_split.cpp [hunk 1]
```diff
@@ -4,37 +4,36 @@
 
 #include <vector>
 
-#include "single_layer_tests/variadic_split.hpp"
+#include "single_op_tests/variadic_split.hpp"
 #include "common_test_utils/test_constants.hpp"
 
-using namespace LayerTestsDefinitions;
+using namespace ov::test;
```
Comment (olpipi) [inline src/plugins/intel_cpu/tests/functional/shared_tests_instances/single_layer_tests/variadic_split.cpp:10]: You don't need whole ov::test here
"using ov::test::VariadicSplitLayerTest" is enough

File: src/tests/functional/shared_test_classes/src/single_op/variadic_split.cpp [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    int64_t axis;
+    std::vector<size_t> num_splits;
+    std::vector<InputShape> input_shapes;
+    ov::element::Type model_type;
+    std::tie(num_splits, axis, model_type, input_shapes, targetDevice) = this->GetParam();
+
+    init_input_shapes(input_shapes);
+
+    auto param = std::make_shared<ov::op::v0::Parameter>(model_type, inputDynamicShapes.front());
+    auto VariadicSplit = std::dynamic_pointer_cast<ov::op::v1::VariadicSplit>(ngraph::builder::makeVariadicSplit(param, num_splits,
```
Comment (olpipi) [inline src/tests/functional/shared_test_classes/src/single_op/variadic_split.cpp:48]: I suppose we should not use builder functions like ngraph::builder::makeVariadicSplit It just create 3 nodes inside it.
Could you please replace it with explicit nodes creation?

File: src/tests/functional/shared_test_classes/src/single_op/variadic_split.cpp [hunk 3]
```diff
@@ -0,0 +1,54 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "shared_test_classes/single_op/variadic_split.hpp"
+#include "ngraph_functions/builders.hpp"
```
Comment (olpipi) [inline src/tests/functional/shared_test_classes/src/single_op/variadic_split.cpp:6]: Remove #include "ngraph_functions/builders.hpp"
Please don't use builders unless it's necessary. We plan to remove most of them

Comment (vurusovs) [reply src/tests/functional/shared_test_classes/src/single_op/variadic_split.cpp]: Thanks, done!

File: src/tests/functional/shared_test_classes/src/single_op/variadic_split.cpp (hunk 2 above)
Comment (vurusovs) [reply src/tests/functional/shared_test_classes/src/single_op/variadic_split.cpp]: Thanks, done!

File: src/plugins/intel_cpu/tests/functional/shared_tests_instances/single_layer_tests/variadic_split.cpp (hunk 1 above)
Comment (vurusovs) [reply src/plugins/intel_cpu/tests/functional/shared_tests_instances/single_layer_tests/variadic_split.cpp]: Thanks, done!

File: src/tests/functional/shared_test_classes/include/shared_test_classes/single_op/variadic_split.hpp [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+#include <vector>
+
+#include "shared_test_classes/base/ov_subgraph.hpp"
+
+namespace ov {
+namespace test {
+typedef std::tuple<std::vector<size_t>,      // Num splits
+                   int64_t,                  // Axis
+                   ov::element::Type,        // Model type
+                   std::vector<InputShape>,  // Input shapes
```
Comment (iefode) [inline src/tests/functional/shared_test_classes/include/shared_test_classes/single_op/variadic_split.hpp:19]: Where is dynamic shapes? We supported dynamism In new base class

Comment (olpipi) [reply src/tests/functional/shared_test_classes/include/shared_test_classes/single_op/variadic_split.hpp:19]: using InputShape = std::pair<ov::PartialShape, std::vector<ov::Shape>>;
So this is what required

**Review by iefode — APPROVED** [review]

**Review by olpipi — APPROVED** [review]

**Review by ilyachur — APPROVED** [review]

**Pull request merged**